    @Test
    public void testWarnAssert() throws InterruptedException {
        LOG.info("An Info message");
        LOG.warn("A warn message");

        try {
            MockLogAppender.assertNotGreaterOrEqual(MockLogger.LOG_LEVEL_WARN);
        } catch (final AssertionFailedError e) {
            return;
        }

        fail("Did not receive expected AssertionFailedError from MockLogAppender.assertNotGreatorOrEqual");
    }

